### ISSUE TITLE
Infusions when importing

### ISSUE THREAD

**Issue by RbbrBootz** [MrPrimate/ddb-importer#302]
Good afternoon

I noticed that when I imported an artificer from DnDB to Foundry, it did give the final AC of an infused armour correctly, but did not show the bonus in the summary.  I went to check the Passive effects and it was all there.  I manually re-created the effect in Foundry and it was all showing properly.  
What happened when importing (tried it twice)
![DNDB Import AC summary](https://user-images.githubusercontent.com/34260597/217050459-a9cfae8e-5271-467e-9331-78937ea40499.PNG)
What happened when manually added the infusion (tried it on a manually created character and erased the import and manually created the infusion on the imported character)
![Standard infused AC summary](https://user-images.githubusercontent.com/34260597/217050462-4e05aa7d-143f-471b-b1de-e8b84d0b8ff4.PNG)
![DNDB Import with manual infusion AC summary](https://user-images.githubusercontent.com/34260597/217051531-6a69f5e7-4b2e-4d77-bc93-ab0dfe7444c7.PNG)

Foundry client V10 Build 291
Windows 10 
  Version 10.0.19045 Build 19045
DnD 5e 2.1.4
Active modules 8

**Comment by MrPrimate** [comment]
Please provide a character link, and also versions of DAE (and Midi-QoL if using). This maybe a DAE problem, the recent releases have been applying effects in duplicate etc.

**Comment by RbbrBootz** [comment]
OK, I am unfamiliar with DAE so I will need some guidance there.
Here is the link to the character
https://www.dndbeyond.com/characters/87400808
Thanks for the quick response

**Comment by MrPrimate** [comment]
You need to have the module Dynamic Active Effects (also known as DAE) for the infusion effects to work properly.

**Comment by RbbrBootz** [comment]
OK.  I installed it.  I deleted the character completely.  I enabled the
DAE module.  But then ran into these two issues.
[image: image.png]
I tried updating it in the module setup outside of the Game World.
This is what happened inside the game.
[image: image.png]
At least it's counting, but it's counting twice.
[image: image.png]
I am assuming we'll have to wait until DAE is compatible with the version
of Foundry I have (291 vs its 286 compatibility).  I can delete the second
effect, so no worries there.  I'll just have to remember when I have the
players import their characters.

Thanks a bunch for your help!

O
